doc/yjit/yjit.md

```diff
@@ -186,19 +186,18 @@ ruby 3.3.0dev (2023-01-31T15:11:10Z master 2a0bf269c9) +YJIT dev [x86_64-darwin2
 
 We have collected a set of benchmarks and implemented a simple benchmarking harness in the [yjit-bench](https://github.com/Shopify/yjit-bench) repository. This benchmarking harness is designed to disable CPU frequency scaling, set process affinity and disable address space randomization so that the variance between benchmarking runs will be as small as possible. Please kindly note that we are at an early stage in this project.
 
-### Performance Tips
+## Performance Tips
```

Comment:
```suggestion
## Code Optimization Tips
```

I think we should rename this and move it below the production performance tips, because people seem not very likely to actually modify their existing Ruby code.

Reply:
applied the rename in https://github.com/ruby/ruby/pull/8472/commits/e972db4aa3ef32c10ab085bd5abf1baea8de2b62 and reordered them in https://github.com/ruby/ruby/pull/8472/commits/89428d0416bf57f7626fa4ef14582ecb93526015.